src/main/java/forestry/arboriculture/genetics/TreeDefinition.java

```diff
@@ -973,9 +974,12 @@ protected void registerMutations() {
 		IGermlingModelProvider germlingIconProvider = ModelProviderFactory.create(woodType, uid, leafIconProvider);
 		IWoodProvider woodProvider = WoodProviderFactory.create(woodType);
 
-		IAlleleTreeSpeciesBuilder speciesBuilder = TreeManager.treeFactory.createSpecies(uid, unlocalizedName, "Sengir", unlocalizedDescription, dominant, branch.getBranch(), binomial, Constants.MOD_ID, leafIconProvider, germlingIconProvider, woodProvider, this);
+		ILeafProvider leafProvider = new LeafProvider();
+
+		IAlleleTreeSpeciesBuilder speciesBuilder = TreeManager.treeFactory.createSpecies(uid, unlocalizedName, "Sengir", unlocalizedDescription, dominant, branch.getBranch(), binomial, Constants.MOD_ID, leafIconProvider, germlingIconProvider, woodProvider, this, leafProvider);
 		setSpeciesProperties(speciesBuilder);
 		this.species = speciesBuilder.build();
+		leafProvider.init(this.species);

```

Comment:
You don't have to call `init` manually it gets called in the `build` method of the species. https://github.com/ForestryMC/ForestryMC/blob/36ccd5c3e1f1aa3f600bbcdd6111b2bd090c64ca/src/main/java/forestry/arboriculture/genetics/alleles/AlleleTreeSpecies.java#L87-L92